backend/src/test/java/edonymyeon/backend/post/integration/PostIntegrationTest.java

```diff
@@ -72,48 +63,57 @@ void setUp() {
                 .when()
                 .post("/posts")
                 .then()
-                .statusCode(HttpStatus.UNAUTHORIZED.value());
+                .extract();
+
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.UNAUTHORIZED.value());
     }
 
     @Test
     void 본인이_작성한_게시글_삭제_가능_테스트() {
-        final Member member = memberTestSupport.builder()
-                .email("email")
-                .password("password")
-                .build();
+        final Member member = 사용자를_하나_만든다();
+        final ExtractableResponse<Response> 게시글_생성_결과 = 게시글을_하나_만든다(member);
 
-        final ExtractableResponse<Response> 게시글_생성_요청_결과 = RestAssured.given()
+        final String location = 게시글_생성_결과.header("location");
+        final long 게시글_id = Long.parseLong(location.split("/")[2]);
+
+        RestAssured.given()
                 .auth().preemptive().basic(member.getEmail(), member.getPassword())
-                .multiPart("title", "this is title")
-                .multiPart("content", "this is content")
-                .multiPart("price", 1000)
-                .multiPart("images", 이미지1, MediaType.IMAGE_JPEG_VALUE)
-                .multiPart("images", 이미지2, MediaType.IMAGE_JPEG_VALUE)
                 .when()
-                .post("/posts")
+                .delete("/posts/" + 게시글_id)
                 .then()
-                .statusCode(HttpStatus.CREATED.value())
-                .extract();
+                .statusCode(HttpStatus.NO_CONTENT.value());
+    }
+
+    @Test
+    void 본인이_작성하지_않은_게시글_삭제_불가능_테스트() {
+        final Member member = 사용자를_하나_만든다();
+        final ExtractableResponse<Response> 게시글_생성_요청_결과 = 게시글을_하나_만든다(member);
 
         final String location = 게시글_생성_요청_결과.header("location");
         final long 게시글_id = Long.parseLong(location.split("/")[2]);
 
+        final Member otherMember = memberTestSupport.builder()
+                .email("other")
+                .password("password")
+                .build();
+
         RestAssured.given()
-                .auth().preemptive().basic(member.getEmail(), member.getPassword())
+                .auth().preemptive().basic(otherMember.getEmail(), otherMember.getPassword())
                 .when()
                 .delete("/posts/" + 게시글_id)
                 .then()
-                .statusCode(HttpStatus.NO_CONTENT.value());
+                .statusCode(HttpStatus.FORBIDDEN.value());
     }
 
-    @Test
-    void 본인이_작성하지_않은_게시글_삭제_불가능_테스트() {
-        final Member member = memberTestSupport.builder()
+    private Member 사용자를_하나_만든다() {
+        return memberTestSupport.builder()
                 .email("email")
                 .password("password")
                 .build();
+    }
```

Comment:
```suggestion
    private Member 사용자를_하나_만든다() {
        return memberTestSupport.builder()
                .build();
    }
```
이렇게 쓰면 범용성이 넓어질것 같아용